controller/formRoutes.js

```diff
@@ -0,0 +1,48 @@
+var model = require('../seqdb/db');
+var formidable = require('formidable');
+var path = require('path');
+var fs = require("fs");
+
+module.exports = {
+    attendance: (req, res, next) => {
+        var form = new formidable.IncomingForm();
+        form.parse(req, function(err, field, file) {
+            var oldpath = file.file.path;
+            var newpath = './uploads/' + file.file.name;
```

Comment:
u have put all code in controller. it first rule of nodejs is never put all code in controller rather user services, models, helper.